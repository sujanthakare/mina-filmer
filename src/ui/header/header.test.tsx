import { MemoryRouter } from 'react-router-dom';

import { render } from '@testing-library/react';

import Header from './header';

describe('Header', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
