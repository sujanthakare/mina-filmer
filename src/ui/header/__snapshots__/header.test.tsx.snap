// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-1u91wc6-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-18 css-1i2jdp9-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-11byt1f-MuiTypography-root"
      >
        Mina Filmer
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to Search"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/"
        tabindex="0"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="SearchRoundedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27c1.2-1.4 1.82-3.31 1.48-5.34-.47-2.78-2.79-5-5.59-5.34-4.23-.52-7.79 3.04-7.27 7.27.34 2.8 2.56 5.12 5.34 5.59 2.03.34 3.94-.28 5.34-1.48l.27.28v.79l4.25 4.25c.41.41 1.08.41 1.49 0 .41-.41.41-1.08 0-1.49L15.5 14zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to favorites"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/favorites"
        tabindex="0"
      >
        Favorites
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to watch later"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/watch-later"
        tabindex="0"
      >
        Watch Later
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
  </div>
</div>
`;

exports[`header should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-1u91wc6-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-18 css-1i2jdp9-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-11byt1f-MuiTypography-root"
      >
        Mina Filmer
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to Search"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/"
        tabindex="0"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="SearchRoundedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27c1.2-1.4 1.82-3.31 1.48-5.34-.47-2.78-2.79-5-5.59-5.34-4.23-.52-7.79 3.04-7.27 7.27.34 2.8 2.56 5.12 5.34 5.59 2.03.34 3.94-.28 5.34-1.48l.27.28v.79l4.25 4.25c.41.41 1.08.41 1.49 0 .41-.41.41-1.08 0-1.49L15.5 14zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to favorites"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/favorites"
        tabindex="0"
      >
        Favorites
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <a
        aria-label="Navigate to watch later"
        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        href="/watch-later"
        tabindex="0"
      >
        Watch Later
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </a>
    </div>
  </div>
</div>
`;
